src/Akka.Persistence.Sql/Journal/Dao/BaseByteArrayJournalDao.cs

```diff
@@ -112,22 +112,37 @@ public async Task<IImmutableList<Exception>> AsyncWriteMessages(
 
         public async Task Delete(string persistenceId, long maxSequenceNr)
         {
+            long maxMarkedDeletion;
+
+            // no need to use transaction
+            await using (var connection = ConnectionFactory.GetConnection())
+            {
+                maxMarkedDeletion = await MaxMarkedForDeletionMaxPersistenceIdQuery(connection, persistenceId, maxSequenceNr).FirstOrDefaultAsync(ShutdownToken);
+            }
```

Comment:
Retrieve the highest affected sequence number that is affected by the delete operation. No need to use transaction, this is strictly a read operation.